core-java-modules/core-java-lang-2/src/test/java/com/baeldung/methodmultiplereturnvalues/MultipleReturnValuesUsingArraysUnitTest.java

```diff
@@ -0,0 +1,47 @@
+package com.baeldung.methodmultiplereturnvalues;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
```

Comment:
You're using JUnit5 assertions with JUnit4 tests - this is confusing. Please choose which you're using and use consistently.

Reply:
done